features/chat_triggers.lua

```diff
@@ -100,8 +100,11 @@ local function mentions(event)
                         success = true
                         break
                     end
+
                     p.print(prefix .. game.get_player(event.player_index).name .. ' mentioned you!', Color.yellow)
                     p.play_sound {path = 'utility/new_objective', volume_modifier = 1}
+                    local message = game.get_player(event.player_index).name .. ' mentioned you in the chat!'
```

Comment:
```suggestion
                    local message = player.name .. ' mentioned you in the chat!'
```
And the same for line 104 above.